package.json

```diff
@@ -18,22 +18,22 @@
     "compile-strict": "tsc -p .",
     "compile": "ts-node -P ./scripts ./scripts npm-compile-all npm-compile-strict",
     "fix": "gts fix",
-    "prepare": "npm run compile"
+    "prepare": "ts-node -P ./scripts ./scripts npm-clean npm-compile"
   },
   "files": [
-    "build/*.js",
-    "build/*.d.ts",
     "build/src/*.js",
+    "build/src/config.d.ts",
     "build/src/constants.d.ts",
-    "build/src/trace-labels.d.ts",
+    "build/src/index.d.ts",
     "build/src/plugin-types.d.ts",
+    "build/src/trace-labels.d.ts",
     "build/src/plugins/*.js",
     "doc",
     "CHANGELOG.md",
     "LICENSE",
     "README.md"
   ],
-  "types": "build/index.d.ts",
+  "types": "build/src/index.d.ts",
```

Comment:
While you're in here, move this next to the main so that changes to one are more visibly caught